recipes-kernel/linux/linux-hikey_4.5.bb

```diff
@@ -0,0 +1,17 @@
+require linux-hikey.inc
+
+DESCRIPTION = "Hisilicon 4.5 Kernel"
+
+PV = "4.5+git${SRCPV}"
+SRCREV_kernel="${AUTOREV}"
+
+SRC_URI = "git://github.com/xin3liang/linux;branch=hikey-tracking-integration-devel-drm;name=kernel \
+           file://defconfig  \
```

Comment:
Defconfig is actually part of this patch, the change is just too large to be displayed by the github website.